tremor-script/src/ast/helper.rs

```diff
@@ -61,7 +61,7 @@ impl Warning {
 #[derive(Default, Debug, Clone, Serialize, PartialEq)]
 pub struct Scope<'script> {
     /// Module of the scope
-    pub(crate) modules: std::collections::HashMap<String, module::Index>,
+    pub(crate) modules: std::collections::BTreeMap<String, module::Index>,
```

Comment:
No objection to the BTreeMap, just curious for a learning perspective why choose the btree map over the hashmap?

Reply:
The reason for this is that iterating over it will do it in alphabetical order. This leads to alphabetically ordered docs.

Reply:
I would prefer insertion order, but this was the quickest i did come up with.

Reply:
Nice insight <3!

Reply:
> I would prefer insertion order, but this was the quickest i did come up with.

could this work?
```rust
Vec<(String, module::Index)>
```